plugin/storage/es/mappings/jaeger-sampling-7.json

```diff
@@ -0,0 +1,21 @@
+{
+  "index_patterns": "*jaeger-sampling-*",
+  {{- if .UseILM }}
+  "aliases": {
+    "{{ .IndexPrefix }}jaeger-sampling-read" : {}
+  },
+  {{- end }}
+  "settings":{
+    "index.number_of_shards": {{ .Shards }},
+    "index.number_of_replicas": {{ .Replicas }},
+    "index.mapping.nested_fields.limit":50,
+    "index.requests.cache.enable":true
```

Comment:
Are you sure about this? The nature of throughput is such that you always want to read & return the latest, I don't think caching is productive for it.